subjects/printfirsthalf/README.md

```diff
@@ -0,0 +1,45 @@
+## printfirsthalf
+
+### Instructions
+
+Write a function called `PrintFirstHalf()` that takes a `string` as an argument and prints the first half of a `string` followed by a newline `\n`.
+
+- If the length of the `string` is odd, round it down.
+- If the `string` is empty, return `Invalid Output` followed by a newline `\n`.
+- If the `string` length is equal to one, return the `string` followed by a newline `\n`.
```

Comment:
Print, or return?

Reply:
return

Reply:
If the exercise name is called `printfirsthalf` and you then ask to return it is not coherent. 
The name has `print` on it.
Line 5 says that it `prints`.
Line 8 says to return.
Line 9 says to return.

You see what I mean now? :slightly_smiling_face: 

It's misleading. It should be consistent.